pkg/command/plugin_group.go

```diff
@@ -28,6 +28,8 @@ var (
 	showNonMandatory bool
 )
 
+const groupSearchShowDetailsMsg = "Note: to view all plugin group versions available, use 'tanzu plugin group search --show-details'"
```

Comment:
Nit: all other notes start their sentence with an uppercase letter.
(Except https://github.com/vmware-tanzu/tanzu-cli/blob/af5d92dce7321228bfd913e15976edcaedb2f0d8/pkg/config/config.go#L33 and https://github.com/vmware-tanzu/tanzu-cli/blob/af5d92dce7321228bfd913e15976edcaedb2f0d8/pkg/config/config.go#L51)

Reply:
Thanks for noticing! Fixed.